ssr/src/page/icpump/ai.rs

```diff
@@ -206,7 +207,30 @@ pub fn ICPumpAiToken(details: TokenListItem) -> impl IntoView {
         <a
             href=details.link
             class="text-xs w-full p-2 flex gap-2 border border-gray-900 bg-transparent hover:bg-white/10 active:bg-white/5">
-          <img src=details.logo class="w-[5.5rem] shrink-0 h-[5.5rem]" />
+          <div class="relative">
+            <img
+                src=details.logo
+                class=move || {
+                    let mut classes = "w-[5.5rem] shrink-0 h-[5.5rem]".to_string();
+                    if details.is_nsfw {
+                        classes.push_str(" blur-md");
+                    }
+                    classes
+                }
+            />
+            {move || if details.is_nsfw {
```

Comment:
If the extra `<div></div>` is not intentional or doesnt have any significiant eg css as dependency etc, Use `<Show when=condition/>` instead of this it will minimize the need of <div></div> in the else block

example:
```jsx
            <Show when=move || authorized().map(|(a, _)| a).unwrap_or_default()>
                <YoutubeUploadInner canisters=cans_s.get_value() url=url_s.get_value() />
            </Show>
```